daemon/main.c

```diff
@@ -693,6 +693,28 @@ static void get_system_timezone(void) {
         timezone = "unknown";
 
     netdata_configured_timezone = config_get(CONFIG_SECTION_GLOBAL, "timezone", timezone);
+
+    //get the utc offset
+    //could be merged with the above strftime call
+    //Note: This will need an agent restart to get new offset on time change (dst, etc).
+    {
+        time_t t;
+        struct tm *tmp, tmbuf;
+        char helper[6];
+
+        t = now_realtime_sec();
+        tmp = localtime_r(&t, &tmbuf);
+
+        if (tmp != NULL) {
+            if(strftime(helper, 6, "%z", tmp) == 0)
+                netdata_utc_offset = strdupz("+0000");
+            else
+                netdata_utc_offset = strdupz(helper);
+        }
+
+        info ("[%ld]", tmbuf.tm_gmtoff);
```

Comment:
please, add more information in this info message to simplify the identification of the value. Netdata stores a lot of information inside the `error.log`.

Reply:
Hi Thiago, sorry for the mess... this is changing at this moment, so that piece of code will go away.